src/replication.c

```diff
@@ -5177,6 +5180,8 @@ void failoverCommand(client *c) {
     
     server.force_failover = force_flag;
     server.failover_state = FAILOVER_WAIT_FOR_SYNC;
+    /* Cancel all ASM tasks when starting failover */
+    clusterAsmCancel(NULL, "failover requested");
```

Comment:
i was thinking we can not allow failover command in cluster mode, but there is possibility we allow it in cluster plugin

Reply:
```
Ozan Tezcan
Out of curiosity, do you allow FAILOVER command?

Joshua Hershberg
Yes b/c we do not support failover within the clustering implementation yet
```

good catch!

Reply:
yes, looks like we use it (or planning to use it)